db/models/user.js

```diff
@@ -1,12 +1,12 @@
 const mongoose = require("mongoose");
+const Schema = require('mongoose').Schema
+const bcrypt = require('bcrypt');
 
-const Schema = mongoose.Schema;
-
-const UserSchema = new Schema({
+const UserSchema = new mongoose.Schema({
     email: {
         type: String,
         unique: true,
-        match: [/.+@.+\..+/, "Please enter a validate email"]
+         match: [/.+@.+\..+/, "Please enter a validate email"]
```

Comment:
Typo- should be "Please enter a *valid* email"

Reply:
i did changed it. DB-Schema stuff coming from Ian ;)

Reply:
I'm happy that this has been addressed